ui/src/org/theGraphApi.ts

```diff
@@ -0,0 +1,201 @@
+import * as apolloCore from "@apollo/client/core";
+import * as ethers from "ethers";
+import * as multihash from "multihashes";
+import * as svelteStore from "svelte/store";
+
+import type * as project from "ui/src/project";
+
+import * as error from "ui/src/error";
+import * as ethereum from "ui/src/ethereum";
+import * as urn from "ui/src/urn";
+import * as wallet from "ui/src/wallet";
+
+const createApolloClient = (uri: string): apolloCore.ApolloClient<unknown> => {
+  return new apolloCore.ApolloClient({
+    uri,
+    cache: new apolloCore.InMemoryCache(),
+    defaultOptions: {
+      query: {
+        fetchPolicy: "no-cache",
+      },
+    },
+  });
+};
+
+const gnosisSubgraphClient = (): apolloCore.ApolloClient<unknown> => {
+  const walletStore = svelteStore.get(wallet.store);
+  let uri;
+  switch (walletStore.environment) {
+    case ethereum.Environment.Local:
+      throw new error.Error({
+        code: error.Code.FeatureNotAvailableForGivenNetwork,
+        message: "Orgs are not available on the Local testnet.",
+      });
+    case ethereum.Environment.Ropsten:
+      uri =
+        "https://api.thegraph.com/subgraphs/name/radicle-dev/gnosis-safe-ropsten";
+      break;
+    case ethereum.Environment.Rinkeby:
+      uri =
+        "https://api.thegraph.com/subgraphs/name/radicle-dev/gnosis-safe-rinkeby";
+      break;
+    case ethereum.Environment.Mainnet:
+      uri = "https://api.thegraph.com/subgraphs/name/radicle-dev/gnosis-safe";
+      break;
+  }
+
+  return createApolloClient(uri);
+};
+
+const orgsSubgraphClient = () => {
+  const walletStore = svelteStore.get(wallet.store);
+  let uri;
+  switch (walletStore.environment) {
+    case ethereum.Environment.Local:
+      throw new error.Error({
+        code: error.Code.FeatureNotAvailableForGivenNetwork,
+        message: "Orgs not available on the Local testnet.",
```

Comment:
```suggestion
        message: "Orgs is not available on the Local testnet.",
```

Reply:
Shouldn't it be "are"?

Reply:
I thought about both. The reason I went with "is" is because Orgs in this case refers the feature, which is a singular noun, and if it says "Orgs are not available…" that's not really true because Orgs are still available in Upstream like… generally… anyway, maybe we should ditch this and write something different since this is confusing. How about:

You can't create an org while on the local testnet.

Reply:
Ok, going with "is" because the context is not org creation.
f261dbaacb12e687c17d1505d8a845f5612a1869